addons/web/static/src/js/views/pivot/pivot_model.js

```diff
@@ -457,16 +457,24 @@ var PivotModel = AbstractModel.extend({
      * @param {any} col_id
      * @param {any} measure
      * @param {any} descending
+     * @param {string} dataType
```

Comment:
no explanation about this. Also, as far as I can tell, it is optional.

`     * @param {'data'|'comparisonData'|'variation'} [dataType]
`